version/version.go

```diff
@@ -1,5 +1,24 @@
 package version
 
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
 // Version specifies the current version of Draupnir
 // This value is injected in at compile time (see the Makefile)
 var Version string
+
+// ParseSemver extracts the major minor and patch level versions from a version string.
+func ParseSemver(version string) (int, int, int, error) {
+	if !regexp.MustCompile("^\\d+\\.\\d+\\.\\d+$").Match([]byte(version)) {
```

Comment:
You can use `MatchString` here, then you don't need to convert `version` to a byte slice